.github/workflows/authorChecklist.yml

```diff
@@ -7,8 +7,7 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
Also, we could help reduce noise and unnecessary runs by being more specific about the [activity types that trigger this workflow](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request):

```yaml
on:
  pull_request:
    types: [opened, edited, reopened]

```

Reply:
I think these are the only activity types that indicate any possible change in the OP / PR description

Reply:
I think we also need `synchronize`

Reply:
no, scratch that. We only care about when the OP changes. I missed that.